src/styles/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc(7 * 100px + 10px * 2);
```

Comment:
What are 7, 100px and other magic numbers? Would be great to create variables for them